presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java

```diff
@@ -174,15 +175,31 @@ public class TestExpressionInterpreter
             .build());
 
     private static final SqlParser SQL_PARSER = new SqlParser();
-    private static final Metadata METADATA = MetadataManager.createTestMetadataManager();
-    private static final TestingRowExpressionTranslator TRANSLATOR = new TestingRowExpressionTranslator(METADATA);
+    private static final TestingPrestoServer server;
+
+    static {
+        try {
+            server = new TestingPrestoServer();
+        }
+        catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
This is a bit heavyweight, can we refactor to avoid using a `TestingPrestoServer`?